packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInputSlot.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+  import VtmnTextInput from '../VtmnTextInput.svelte';
+</script>
+
+<VtmnTextInput id="textinput" value="Value" {...$$restProps}>
```

Comment:
That should be enough

```suggestion
<VtmnTextInput {...$$restProps}>
```